accounts-db/src/accounts_db.rs

```diff
@@ -7479,6 +7484,13 @@ impl AccountsDb {
             .cloned()
     }
 
+    /// Get all incremental accounts hashes
```

Comment:
Not sure it requires any special "markup", but currently, we only plan to use this function (and the other new one) to get extra information before we panic, right? I guess it is a separate file AND crate, so these have to be `pub`